rca/project_styleguide/templates/patterns/pages/home/home_page.html

```diff
@@ -13,29 +13,35 @@
 
         <div class="page__content">
 
-            <section class="section section--top-space-large bg bg--dark">
-
-                <div class="proposition-statement grid">
-                    <div class="proposition-statement__introduction">The world’s number 1 art & design university</div>
-                    <a href="#" class="proposition-statement__link link link--primary link--link link--book">
-                        <span class="link__label">Book an open day</span>
-                        <svg width="12" height="8" class="link__icon"><use xlink:href="#arrow"></use></svg>
-                    </a>
-                </div>
-
-                <div class="section__notch">
-                    <div class="section__notch-fill section__notch-fill--second-col"></div>
-                </div>
+            {% if page.strapline %}
+                <section class="section section--top-space-large bg bg--dark">
+
+                    <div class="proposition-statement grid">
+                        <div class="proposition-statement__introduction">{{ page.strapline }}</div>
+                        {% if page.strapline_cta_url and page.strapline_cta_text %}
+                            <a href="{{ page.strapline_cta_url }}" class="proposition-statement__link link link--primary link--link link--book">
+                                <span class="link__label">{{ page.strapline_cta_text }}</span>
+                                <svg width="12" height="8" class="link__icon"><use xlink:href="#arrow"></use></svg>
+                            </a>
+                        {% endif %}
+                    </div>
 
-            </section>
+                    <div class="section__notch">
+                        <div class="section__notch-fill section__notch-fill--second-col"></div>
+                    </div>
 
-            <section class="section section--top-space bg bg--light">
-                {% include "patterns/organisms/image-video-block/image-video-block.html" with modal=False heading=image_video_block.heading subheading=image_video_block.subheading has_meta=True %}
-
-                <div class="section__notch">
-                    <div class="section__notch-fill section__notch-fill--third-col"></div>
-                </div>
-            </section>
+                </section>
+            {% endif %}
+            {% if transformation_blocks.first %}
```

Comment:
I am bamboozled why can't you just pass one instance from the view if you will only display one.

Reply:
Thanks Tomasz, changed on https://github.com/torchbox/rca-wagtail-2019/pull/119
```
context["transformation_block"] = self.transformation_blocks.select_related(
            "image"
        ).first()```